src/components/SignIn.js

```diff
@@ -30,17 +35,85 @@ function Copyright(props) {
 }
 
 export default function SignIn() {
-  const handleSubmit = (event) => {
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  //update the input states when their value is changed
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({ ...prevData, [name]: value }));
+  };
+
+  const navigate = useNavigate();
+
+  // snackbar
+  const [open, setOpen] = React.useState(true);
```

Comment:
```suggestion
  const [open, setOpen] = React.useState(false);
```

Otherwise, we're starting with the snackbar open but empty:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/cc3d5826-c9c6-4316-b860-39c03fbd3d99)

Reply:
[@JafariM](https://github.com/JafariM) I think this still applies; maybe you fixed it before but it looks like it's still there, maybe it got undone with the last merge or something?